Gemfile.lock

```diff
@@ -254,6 +254,7 @@ GEM
     faraday-net_http (3.3.0)
       net-http
     ffi (1.17.0-arm64-darwin)
+    ffi (1.17.0-x86_64-darwin)
```

Comment:
There are a few changes to this file which I wouldn't expect as part of this PR (we're not changing any gems so I'm not sure why the `Gemfile.lock` has changed).

We should probably remove the file from the PR.